plugin/src/main/kotlin/org/gradle/github/dependency/extractor/GitHubDependencyExtractorPlugin.kt

```diff
@@ -34,64 +33,64 @@ class GitHubDependencyExtractorPlugin : Plugin<Gradle> {
         const val ENV_GITHUB_WORKSPACE = "GITHUB_WORKSPACE"
     }
 
-    internal lateinit var dependencyExtractorServiceProvider: Provider<out DependencyExtractorService>
+    internal lateinit var dependencyExtractorProvider: Provider<out DependencyExtractor>
 
     override fun apply(gradle: Gradle) {
         val gradleVersion = GradleVersion.current()
         // Create the adapter based upon the version of Gradle
         val applicatorStrategy = when {
-            gradleVersion >= GradleVersion.version("6.1") -> PluginApplicatorStrategy.PluginApplicatorStrategy_6_1
+            gradleVersion < GradleVersion.version("8.0") -> PluginApplicatorStrategy.LegacyPluginApplicatorStrategy
```

Comment:
Why use the legacy version for so many versions that will support the newer API?

Reply:
The "latest" strategy only works for Gradle 7.4+, versions 6.1 - 7.3 versions require the deprecated `forUseAtConfigurationTime`.
We might add an intermediate strategy later, but if the legacy strategy works then I don't see a good reason to add the complexity. And using a strategy consistently for a major Gradle version seems like it may cause fewer headaches or confusion in the future.
We'll see...

Reply:
Reasonable rational. Sounds good to me!